dto/src/main/java/dto/TicketRecordDto.java

```diff
@@ -0,0 +1,25 @@
+package dto;
+
+import org.springframework.data.annotation.Id;
+
+import java.sql.Timestamp;
+
+public record TicketRecordDto(@Id Long id, String departCity, String arriveCity, String site, Long price, Timestamp departDate,
+                              Timestamp returnDate, Integer numberOfChanges, Long duration, Long distance) {
+
+    @Override
+    public String toString() {
```

Comment:
Do we need special format of to string instead of the produced by the record featuer?